lib/device-log/ios-crash-log.ts

```diff
@@ -124,10 +124,11 @@ export class IOSCrashLog extends IOSLog<TSerializedEntry, TSerializedEntry> {
   }
 
   private async _gatherFromRealDevice(): Promise<string[]> {
-    if (!this._realDeviceClient || !await this._realDeviceClient.assertExists(false)) {
+    if (!this._realDeviceClient) {
       return [];
     }
 
+    await this._realDeviceClient.assertExists(true);
```

Comment:
This change will start blocking a new session for env that does not have `pyidevice`

Reply:
makes sense. Changed

Reply:
Maybe... `startCapture`'s _listCrashFiles should keep not raising exception, or need to another check in the start?

Reply:
(Noticed 66644eb4f3ad4bfaef3dfdda08bdb424446c15d0)